scripts/e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+export CHROME_HEADLESS=true
+setup_service google-chrome-stable 83.0.4103.61-1
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+aws s3 --quiet --region us-east-1 cp s3://ci-secret-stash/prod/signinwidget/test_credentials ./test_credentials.yaml
```

Comment:
would that make sense to create a JSON or even `.sh` file (instead of yaml) at first place?

Reply:
We could have json or a script. What advantage do you think that gives us over yaml?
If we create a json file, I will have to ask eng-prod to upload it to aws first. And then parse the downloaded json to get the variables. It's a little bit of work. Unless you think having this yaml is an issue, I don't see a need to change?

Reply:
just trying to avoid extra dependencies (pip install yq). that's why I'm curious how `test_credentials.yaml` was created at first place?

Reply:
Oh ok. That makes sense. I probably created this convoluted way in monolith and just migrated it here.

Reply:
@haishengwu-okta - Have removed the yaml file and added a shell script with env vars exported directly. No more dependency on `yq`.